src/index.css

```diff
@@ -1622,6 +1631,13 @@ td:first-child .checkbox label {
   margin: 0!important;
   color: #777777;
 }
+.catalog-table-items {
+  font-weight: normal !important;
+  font-size: 14px !important;
+  padding: 10px 20px 5px 5px !important;
+  margin: 0!important;
+  color: #000000;
+}
```

Comment:
Please look at `theme.js` which is where I think this should exist. Make these two classes style objects and then import the object directly into the catalog page directly. Have a look [here](https://github.com/DataBiosphere/duos-ui/blob/develop/src/pages/access_review/ApplicantInfo.js) where we create constants that contain style information, then set the `style` property directly on the html elements where we want them. This way, if these are pulled into a common theme, other classes can then start using them. Rename the styles to better reflect what they mean, i.e. `lightTable` or something that makes sense to you.